content/concepts/consensus-network/transaction-malleability.md

```diff
@@ -86,35 +72,29 @@ For greater security, these guidelines provide multiple layers of protection.
 
-1. The vulnerable system constructs and signs a transaction without enabling `tfFullyCanonicalSig`.
-
-    Three ways that a transaction may not enable the `tfFullyCanonicalSig` flag are:
-
-    - The system explicitly specifies a `Flags` field that does not have the `tfFullyCanonicalSig` bit enabled.
-    - The transaction is multi-signed and does not explicitly enable the `tfFullyCanonicalSig` flag.
-    - The system omits the `Flags` field from the transaction fields, but uses a non-standard implementation that does not automatically enable `tfFullyCanonicalSig` when signing.
+1. The vulnerable system constructs a multi-signed transaction and collects more than the necessary number of signatures.
 
-    To be vulnerable, the transaction must be signed with an ECDSA key pair. If multi-signed, the transaction must be signed by at least one ECDSA key pair.
-
-    Most likely, the vulnerable transaction uses a fully-canonical signature, but the flags indicate that the transaction would also be valid with a non-fully-canonical one. The transaction may also use `LastLedgerSequence` so that its final outcome is clear in a finite amount of time.
+    If an authorized signer is malicious or irresponsible, the transaction could also be vulnerable if that signer's signature is not included but could be added.
 
 2. The system notes the identifying hash of the vulnerable transaction, submits it to the XRP Ledger network, then begins monitoring for that hash to be included in a validated ledger version.
 
 3. A malicious actor sees the transaction propagating through the network before it becomes confirmed.
 
-4. The malicious actor calculates the alternate signature for the vulnerable transaction.
+4. The malicious actor calculates removes an extra signature from the vulnerable transaction.
 
     Unlike creating a signature for different transaction instructions, this does not require a large amount of computational work. It can be done in much less time than it takes to generate a signature in the first place.
 
-    The modified signature results in a different identifying hash. (You do not have to calculate the hash before you submit to the network, but knowing the hash makes it easier to check the transaction's status later.)
+    Alternatively, an authorized signer whose signature is not already part of the transaction could add their signature to the vulnerable transaction's list of signatures. Depending on the sender's multi-signing settings, this can be in addition to removing other signatures from the transaction.
+
+    The modified list of signatures results in a different identifying hash. (You do not have to calculate the hash before you submit to the network, but knowing the hash makes it easier to check the transaction's status later.)
 
-5. The malicious actor submits the modified (likely non-fully-canonical) transaction to the network.
+5. The malicious actor submits the modified transaction to the network.
```

Comment:
@mDuo13 , another "The x actor".

Reply:
Again, same malicious actor, thus, "the"